drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -181,7 +181,7 @@ func (k *k8sSchedOps) ValidateVolumeSetup(vol *volume.Volume) error {
 
 		containerPaths := getContainerPVCMountMap(p)
 		for containerName, path := range containerPaths {
-			pxMountCheckRegex := regexp.MustCompile(fmt.Sprintf("^(/dev/pxd.+|pxfs.+) on %s.+", path))
+			pxMountCheckRegex := regexp.MustCompile(fmt.Sprintf("^(/dev/pxd.+|\\d+\\.\\d+\\.\\d+\\.\\d+:/var/lib/osd/pxns/.+) on %s.+", path))
```

Comment:
Perhaps a comment to explain to mere mortals what the hell this is supposed to match?